Credit for the idea of Assigning Node Metadata to Pods (#1344)
kyverno/website: content/en/blog/general/assigning-node-metadata-to-pods/index.md

              # https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone
              topology.kubernetes.io/zone: "{{ ZoneLabel }}"
```
### Credits

> Thanks [Abir Sigron](https://github.com/abirsigron) for initiating the innovative idea and contributing to its successful implementation.

Reviewer: ```suggestion
Thanks to [Abir Sigron](https://github.com/abirsigron) for bringing this up in Slack.
```
